docs/glossary/p/precedence.lcdoc

```diff
@@ -7,12 +7,16 @@ Type: glossary
 Description:
 The set of rules that determine what order <operator|operators> are
 <evaluate|evaluated> in, in an <expression> that has more than one
-<operator>. 
+<operator>. Simplified, the rules for numeric operators are as follows:
 
-For example, the * <operator> has higher precedence than the +
-<operator>, so 2 + 3 * 4
-is <evaluate|evaluated> as 2 + (3 * 4), not (2 + 3) * 4.
+1. Operations grouped by parentheses.
+2. Exponentiation.
+3. Multiplication and division, from left to right.
+4. Addition and subtraction, from left to right.
+
+For example, the \* <operator> has higher precedence than the + 
+<operator>, so 2 + 3 \* 4
+is <evaluate|evaluated> as 2 + (3 \* 4), not (2 + 3) \* 4.
```

Comment:
For the first occurrence, does it work to actually link to the asterisk operator using `<*>`?